File: frameworks/PHP/laravel/routes/web.php
<?php

use App\Http\Controllers\Controller;
use Illuminate\Support\Facades\Route;

Route::get('/json', [Controller::class, 'json']);
Route::get('/db', [Controller::class, 'db']);
Route::get('/queries/{queries?}', [Controller::class, 'queries']);
Route::get('/fortunes', [Controller::class, 'fortunes']);
Route::get('/updates/{queries?}', [Controller::class, 'updates']);
Route::get('/plaintext', [Controller::class, 'plaintext']);

File: frameworks/Go/goravel/src/gin/routes/web.go
package routes

import (
	"github.com/goravel/framework/facades"

	"goravel/app/http/controllers"
)

func Web() {
	testController := controllers.NewTestController()
	facades.Route().Get("/plaintext", testController.Plaintext)
	facades.Route().Get("/json", testController.JSON)
	facades.Route().Get("/db", testController.DB)
	facades.Route().Get("/queries", testController.Queries)
	facades.Route().Get("/update", testController.Update)
	facades.Route().Get("/fortunes", testController.Fortunes)
	facades.Route().Get("/cached-worlds", testController.CacheQueries)

}
